pkg/graveler/model.go

```diff
@@ -17,6 +17,7 @@ const (
 	settingsPrefix         = "settings"
 	addressesPrefix        = "link-addresses"
 	importsPrefix          = "imports"
+	repoMetadataPrefix     = "metadata"
```

Comment:
Will this work?

```suggestion
	repoMetadataPrefix     = "metadata/repo"
```

Or if not,
```suggestion
	repoMetadataPrefix     = "repos-metadata"
```

One day we'll want whole-installation metadata, and maybe per-user metadata etc.

Reply:
This information is per repo partition - will change to repo_metadata